File: reactive-test/src/main/java/matthias/reactive_test/shops_reactive/ShopEntity.java
package matthias.reactive_test.shops_reactive;

import lombok.Getter;
import lombok.Setter;
import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Table;

import java.util.UUID;

import static java.util.UUID.randomUUID;

@Getter
@Setter
@Table(name = "purchase_shop")
public class ShopEntity {

    @Id
    private UUID id = randomUUID();

    private String name;
}

File: fintracker-backend/src/main/kotlin/com/matthias/fintracker/shop/ShopEntity.kt
package com.matthias.fintracker.shop

import com.matthias.fintracker.common.jpa.BaseEntity
import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Table
import java.util.*

@Entity
@Table(name = "purchase_shop")
class ShopEntity : BaseEntity {

    constructor() : super()
    constructor(id: UUID?) : super(id)

    @Column(columnDefinition = "citext", nullable = false)
    lateinit var name: String
}
